kiali-server/templates/role-controlplane.yaml

```
@@ -14,4 +14,14 @@ rules:
   verbs:
   - list
 {{- end }}
+- apiGroups: [""]
+  resourceNames:
+  - cacerts
+  - istio-ca-secret
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
```

This section isn't using the new config setting. This instead should be:
```suggestion
{{- if .Values.kiali_feature_flags.certificates_information_indicators.enabled }}
- apiGroups: [""]
  resourceNames: {{ .Values.kiali_feature_flags.certificates_information_indicators.secrets }}
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
{{- end }}
```
I'm not exactly sure of the syntax to get that list value set in `resourceNames` but you get the idea.